src/createLoadableVisibilityComponent.js

```diff
@@ -95,7 +77,7 @@ function createLoadableVisibilityComponent (args, {
         return <div
           style={{display: 'inline-block', minHeight: '1px', minWidth: '1px'}}
           className={this.props.className}
-          ref={this.attachRef}
+          ref="loading"
```

Comment:
Can you revert this change? String refs have been deprecated - https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs